### PR TITLE
Topics: Removed topics from A-N, R, S, V, W #5

### PR DESCRIPTION
**Pull request by clmedders**
## Summary

> [!NOTE]
> This is the first removal PR that should be merged into `topics-removal-integration`

Removed the following **135** topics:

- [x] abmc
- [x] administrative-office-of-the-united-states-courts
- [x] administrative-office-of-the-united-states-courts
- [x] american-battle-monuments-commission
- [x] american-red-cross
- [x] analytics-usa-gov
- [x] architect-of-the-capitol
- [x] arlington-national-cemetery
- [x] authentication
- [x] bbg
- [x] blog
- [x] bls
- [x] broadcasting-board-of-governors
- [x] bureau-of-justice-statistics
- [x] bureau-of-reclamation
- [x] cdc
- [x] census
- [x] centers-for-disease-control-and-prevention
- [x] central-intelligence-agency
- [x] cfpb
- [x] cia
- [x] cms
- [x] cncs
- [x] commerce
- [x] consumer-financial-protection-bureau
- [x] consumer-product-safety-commission
- [x] corporation-for-national-and-community-service
- [x] court-services-and-offender-supervision-agency
- [x] csosa
- [x] darpa
- [x] defense-advanced-research-projects-agency
- [x] defense-finance-and-accounting-services
- [x] department-of-commerce
- [x] dfas
- [x] dhs
- [x] digitalgov-summit
- [x] dod
- [x] doe
- [x] doi
- [x] doj
- [x] dol
- [x] dot
- [x] education
- [x] epa
- [x] ex-im-bank
- [x] executive-office-of-the-president
- [x] export-import-bank-of-the-united-states
- [x] faa
- [x] fafsa
- [x] fas
- [x] fbi
- [x] fcc
- [x] fda
- [x] fec
- [x] federal-acquisition-service
- [x] federal-aviation-administration
- [x] federal-bureau-of-investigation
- [x] federal-communications-commission
- [x] federal-election-commission
- [x] federal-emergency-management-agency
- [x] federal-railroad-administration
- [x] federal-register
- [x] federal-reserve-board
- [x] federal-student-aid
- [x] federal-trade-commission
- [x] fema
- [x] firstnet
- [x] frb
- [x] fsa
- [x] ftc
- [x] fws
- [x] gao
- [x] go-usa-gov
- [x] hhs
- [x] hrsa
- [x] hud
- [x] ice
- [x] indian-health-service
- [x] internal-revenue-service
- [x] internal-revenue-service
- [x] irs
- [x] itif
- [x] kids-gov
- [x] law-library-of-congress
- [x] library-of-congress
- [x] loc
- [x] mcc
- [x] nara
- [x] nasa
- [x] national-aeronautics-and-space-administration
- [x] national-archives-and-records-administration
- [x] national-cancer-institute
- [x] national-center-for-complementary-and-integrative-health
- [x] national-center-for-telehealth-and-technology
- [x] national-eye-institute
- [x] national-gallery-of-art
- [x] national-highway-traffic-safety-administration
- [x] national-institute-of-standards-and-technology
- [x] national-institute-on-drug-abuse
- [x] national-institutes-of-health
- [x] national-library-of-medicine
- [x] national-science-foundation
- [x] national-telecommunications-and-information-administration
- [x] national-weather-service
- [x] national-zoological-park
- [x] nccih
- [x] nci
- [x] nhtsa
- [x] recaps
- [x] regulation
- [x] resource
- [x] samhsa
- [x] sba
- [x] sec
- [x] sitesusagov
- [x] small-business-innovation-research
- [x] smithsonian-institution
- [x] social-and-behavioral-sciences-team
- [x] ssa
- [x] stat
- [x] technology-transformation-services
- [x] the-united-states-social-security-administration
- [x] training
- [x] transportation-security-administration
- [x] tribal-government
- [x] tsa
- [x] tt
- [x] va
- [x] va-center-for-innovation
- [x] vac
- [x] web-based
- [x] webinar
- [x] white-house
- [x] white-house-office-of-science-and-technology-policy
- [x] world-bank

### Removal Steps

- [x] removed topics starting with in `/content/topics/[topic]`
- [x] removed topic tag in the `topics` front matter field
- [x] archived pages on wayback machine

---

### Dev Checklist

- [x] PR has correct labels
- [x] A11y testing (voice over testing, meets WCAG, run axe tools)
- [x] Code is formatted properly

### CODE REVIEW

**Comment by clmedders** [comment]
@mejiaj when you make it to this can you review `national-telecommunications-and-information-administration` im getting a build error when i remove the index but i cant find a reference for it...

**Comment by clmedders** [comment]
fixed the merge conflicts

File: content/news/2013/03/2013-03-14-nci-quitpal-app.md [hunk 1]
```diff
@@ -7,7 +7,6 @@ topics:
   - mobile
   - mobile-apps
   - mobile-gov
-  - NCI
   - nih
```
Comment (mejiaj) [inline content/news/2013/03/2013-03-14-nci-quitpal-app.md:10]: Is this an agency that we've decided to keep?

File: content/news/2013/03/2013-03-21-american-red-cross-tornado-app.md [hunk 2]
```diff
@@ -5,7 +5,6 @@ title: American Red Cross Tornado App
 summary: The American Red Cross Tornado App gets your family and home ready for a tornado. The app, which is available on the Android and iOS platforms, puts everything you need to know
 topics:
   - mobile
-  - american-red-cross
   - mobile-apps
   - mobile-gov
   - NOAA
```
Comment (mejiaj) [inline content/news/2013/03/2013-03-21-american-red-cross-tornado-app.md:10]: Another agency?

File: content/news/2013/05/2013-05-07-the-new-healthcare-gov-uses-a-lightweight-open-source-tool.md [hunk 3]
```diff
@@ -8,7 +8,6 @@ authors:
 topics:
   - code
   - content-strategy
-  - HHS
   - open-data
   - open-source
   - united-states-department-of-health-and-human-services
```
Comment (mejiaj) [inline content/news/2013/05/2013-05-07-the-new-healthcare-gov-uses-a-lightweight-open-source-tool.md:13]: Marking this agency for documentation.

File: content/news/2013/05/2013-05-16-api-release-kit.md [hunk 4]
```diff
@@ -9,11 +9,6 @@ topics:
...
   - code
   - api
-  - Census
-  - epa
-  - FCC
-  - federal-register
-  - National Weather Service
   - NREL
   - us-department-of-labor
 ---
```
Comment (mejiaj) [inline content/news/2013/05/2013-05-16-api-release-kit.md:14]: More agencies.

**Review by mejiaj — APPROVED** [review]
Looks good to me. 

Found a few issues in PR description and a file that had blank lines [1042b1c81e0ec7bd5bab1e545b940d49cbc19016].

### PR description

1. Description shows `administrative-office-of-the-united-states-courts` twice.
2. `internal-revenue-service` is mentioned twice.
3. Topic `analytics-usa-gov` isn't mentioned.
4. Topic `best-practices` isn't mentioned.


## How I tested

I used a regex search based on description to look for topics and ensure they're not showing:

```
^\s*-\s* (bureau-of-reclamation|cdc|census|centers-for-disease-control-and-prevention|central-intelligence-agency|cfpb|cia|cms|cncs|commerce|consumer-financial-protection-bureau|consumer-product-safety-commission|corporation-for-national-and-community-service|court-services-and-offender-supervision-agency|csosa|darpa|defense-advanced-research-projects-agency|defense-finance-and-accounting-services|department-of-commerce|dfas|dhs|digitalgov-summit|dod|doe|doi|doj|dot|education|epa|ex-im-bank|executive-office-of-the-president|export-import-bank-of-the-united-states|faa|fas|fbi|fcc|fda|fec|federal-acquisition-service|federal-aviation-administration|federal-bureau-of-investigation|federal-communications-commission|federal-election-commission|federal-emergency-management-agency|federal-railroad-administration|federal-register|federal-reserve-board|federal-student-aid|federal-trade-commission|fema|firstnet|frb|fsa|ftc|fws|gao|go-usa-gov|hhs|hrsa|hud|ice|indian-health-service|internal-revenue-service|irs|itif|kids-gov|law-library-of-congress|loc|mcc|nara|nas|nasa|national-aeronautics-and-space-administration|national-archives-and-records-administration|national-cancer-institute|national-center-for-complementary-and-integrative-health|national-center-for-telehealth-and-technology|national-eye-institute|national-gallery-of-art|national-highway-traffic-safety-administration|national-institute-of-standards-and-technology|national-institute-on-drug-abuse|national-institutes-of-health|national-library-of-medicine|national-science-foundation|national-telecommunications-and-information-administration|national-weather-service|national-zoological-park|nccih|nci|nhtsa|recaps|regulation|resource|samhsa|sba|sec|sitesusagov|small-business-innovation-research|smithsonian-institution|social-and-behavioral-sciences-team|ssa|stat|technology-transformation-services|the-united-states-social-security-administration|training|transportation-security-administration|tribal-government|tsa|tt|va|va-center-for-innovation|vac|web-based|webinar|white-house|white-house-office-of-science-and-technology-policy|world-bank)$
```

File: content/news/2013/03/2013-03-14-nci-quitpal-app.md (hunk 1 above)
Comment (ToniBonittoGSA) [reply content/news/2013/03/2013-03-14-nci-quitpal-app.md:10]: @mejiaj  Is in part 2, https://github.com/GSA/digitalgov.gov/pull/7140

File: content/news/2013/05/2013-05-16-api-release-kit.md (hunk 4 above)
Comment (ToniBonittoGSA) [reply content/news/2013/05/2013-05-16-api-release-kit.md:14]: nrel in https://github.com/GSA/digitalgov.gov/pull/7140 
us-department-of-labor in https://github.com/GSA/digitalgov.gov/pull/7152

File: content/news/2013/03/2013-03-21-american-red-cross-tornado-app.md (hunk 2 above)
Comment (ToniBonittoGSA) [reply content/news/2013/03/2013-03-21-american-red-cross-tornado-app.md:10]: Was removed from all files, added to list in initial comment

File: content/news/2013/05/2013-05-07-the-new-healthcare-gov-uses-a-lightweight-open-source-tool.md (hunk 3 above)
Comment (ToniBonittoGSA) [reply content/news/2013/05/2013-05-07-the-new-healthcare-gov-uses-a-lightweight-open-source-tool.md:13]: Also removed, added to checklist above

**Review by ToniBonittoGSA — APPROVED** [review]

**Pull request merged**